packages/nextjs/drizzle.config.ts

```diff
@@ -3,6 +3,13 @@ import { defineConfig } from "drizzle-kit";
 
 dotenv.config({ path: ".env.development" });
 
+const PRODUCTION_DATABASE_HOSTNAME = "cold-resonance";
+
+if (process.env.POSTGRES_URL?.includes(PRODUCTION_DATABASE_HOSTNAME) && !process.argv.includes("--force")) {
+  console.log("\n⚠️ You are pointing to the production database. Use `--force` if you want to continue.\n");
+  process.exit(0);
+}
```

Comment:
For the problem which rinat mentioned what if instead of using the `--flag` we could set the env variable while running the command? 

eg: 

```
ALLOW_PROD_ACTIONS=true yarn drizzle-kit migrate
```

Reply:
I've been thinking about this since Rinat's review.

This option was an improvment (since it simplifies everything) but it has something (as the --force one) that I don't like: people (me) sometime use CTRL+R to search for previous commands (e.g. push / migrate) and if you hit enter, (without noticing / ignore the ENV_VAR) you are f***

So went with another option. Pushing in 2 min

Reply:
```
ALLOW_PROD_ACTIONS=true yarn drizzle-kit migrate
```

Yes, I've been thinking about that solution, but if someone will add `ALLOW_PROD_ACTIONS=true` to `.env`, than, again

> you are f***